pkg/collector/corechecks/servicediscovery/usm/service_test.go

```diff
@@ -666,6 +669,10 @@ func TestExtractServiceMetadata(t *testing.T) {
 				require.Equal(t, tt.expectedGeneratedName, meta.Name)
 				require.Equal(t, tt.expectedAdditionalServices, meta.AdditionalNames)
 				require.Equal(t, tt.ddServiceInjected, meta.DDServiceInjected)
+				require.NotEmpty(t, meta.Source)
+				if len(tt.expectedSource) != 0 {
```

Comment:
Why is the source checked only on some cases?

Reply:
Fixed.